e2e/shortcut-action/shortcut-action.po.ts

```diff
@@ -20,13 +20,13 @@
  * as an Intergovernmental Organization or submit itself to any jurisdiction.
  */
 
-import { element, by } from 'protractor/globals';
+import { element, by, protractor } from 'protractor/globals';
 import { Ng2JsonEditorPage } from '../app.po';
-import {ElementFinder} from 'protractor';
+import { ElementFinder, ElementArrayFinder } from 'protractor';
 
 export class ShortcutActionPage extends Ng2JsonEditorPage {
 
-  getNumberOfChildRowsbyId(id: string) {
+  getNumberOfChildRowsbyId(id: string): webdriver.promise.Promise<number> {
```

Comment:
can we have shorthand for `webdriver.promise.Promise<T>` like `export type WDPromise<T> = webdriver.promise.Promise<T>`. Use it if it works, don't worry if it doesn't.